apps/e2e-test/src/playwright-support/app-specific/element-verification.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from '@playwright/test';
+import { Page } from 'playwright';
+
+import { logger } from '../../logger';
+import { TargetSelector } from '../../utils/selector-mapper';
+import { Screen, Status } from '../../utils/types';
+
+import { getDeviceStatus, getOptions, getViewScreenSize } from './element-read';
+
+export const verifyViewScreenSize = async (
+  page: Page,
+  selector: TargetSelector,
+  screenSize: Screen,
+  negate = false
+): Promise<void> => {
+  logger.trace(`Verify element view size${negate ? ' not' : ''} to be ${screenSize}`);
+  if (negate) {
+    expect(await getViewScreenSize(page, selector)).not.toEqual(screenSize);
+  } else {
+    expect(await getViewScreenSize(page, selector)).toEqual(screenSize);
+  }
+};
+
+export const verifyDeviceStatus = async (
+  page: Page,
+  selector: TargetSelector,
+  status: Status,
+  negate = false
+): Promise<void> => {
+  logger.trace(`Verify device status${negate ? ' not' : ''} to be turned ${status}`);
+  if (negate) {
+    expect(await getDeviceStatus(page, selector)).not.toEqual(status);
```

Comment:
nit: you could lift the `await getDeviceStatus(page, selector));` bit out of the if else, store it in a variable and then use it below in the `expect`

Reply:
Fixed